packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -196,7 +206,8 @@ export const InputBoxButton = (props: InputBoxButtonProps): JSX.Element => {
         onMouseLeave={() => {
           setIsHover(false);
         }}
-        onRenderIcon={() => onRenderIcon(isHover)}
+        // VoiceOver fix: Avoid rerender of DefaultButton above that handles clicking and hovering by keeping rerendering icon separate
```

Comment:
Could you change my old comment to be `VoiceOver fix: Avoid icon from stealing focus when IconButton is double-tapped to send message`?